README.md

```diff
@@ -151,8 +151,8 @@ zone.  This role has a trust relationship with the users account.
 | acmresourcechange\_role\_name | The name to assign the IAM role (as well as the corresponding policy) that allows sufficient permissions to modify ACM (AWS Certificate Manager) resources in the DNS account. | `string` | `"ACMResourceChange"` | no |
 | additional\_remote\_state\_account\_ids | A list of account IDs corresponding to additional accounts that should have permission to assume the role to read this root module's remote state (e.g. ["123456789012"]). | `list(string)` | `[]` | no |
 | additional\_ses\_sendemail\_account\_ids | A list of account IDs corresponding to additional accounts that should have permission to assume the role to send email via AWS SES (e.g. ["123456789012"]). | `list(string)` | `[]` | no |
+| api\_gateway\_zone\_id | The Route 53 hosted zone ID for API Gateways. | `string` | `"Z1UJRXOUMOOFQ8"` | no |
```

Comment:
Remove unnecessary capitalization:
```suggestion
| api\_gateway\_zone\_id | The Route 53 hosted zone ID for API gateways. | `string` | `"Z1UJRXOUMOOFQ8"` | no |
```

Reply:
Fixed in this commit [c14aa64](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/92/commits/c14aa64dca18d92c60eec7aab6b358b685090546)